packages/affiliates/test/e2e/datasets.js

```diff
@@ -173,50 +180,23 @@ describe("Datasets", function() {
       assert(result.length);
     });
   });
-  describe("Synthprices Serializer", function() {
-    it("serializes", async function() {
-      this.timeout(10000);
-      const config = {
-        contract: params.empContracts[0],
-        start: params.startingTimestamp,
-        end: params.startingTimestamp + 1000 * 60 * 60 * 24 * 10
-      };
-      const data = await synthPrices.getHistoricSynthPrices(config.contract.toLowerCase(), config.start, config.end);
-      const result = SynthPrices().serialize(data);
-      assert(result);
-    });
-    it("deserializes", async function() {
-      this.timeout(10000);
-      const config = {
-        contract: params.empContracts[0],
-        start: params.startingTimestamp,
-        end: params.startingTimestamp + 1000 * 60 * 60 * 24 * 10
-      };
-      const data = await synthPrices.getHistoricSynthPrices(config.contract.toLowerCase(), config.start, config.end);
-      const serialized = SynthPrices().serialize(data);
-      const result = await SynthPrices()
-        .deserialize(serialized.split())
-        .toPromise(Promise);
-      assert(result.length);
-    });
-  });
-  describe("Dataset and Mocks", function() {
+  describe("Dataset and Mocks for Dev Mining", function() {
     let path;
     after(function(done) {
       fs.rmdir(basePath, { recursive: true }, done);
     });
     it("saves new set", async function() {
-      this.timeout(10000);
+      this.timeout(100000);
       const config = {
         ...params,
         start: params.startingTimestamp,
         end: params.startingTimestamp + 1000 * 60 * 60 * 24 * 10
       };
       const ds = Dataset(basePath, { queries, coingecko, synthPrices });
-      path = await ds.save("test-set", config);
+      path = await ds.saveDevMining("dev-mining-test", config);
```

Comment:
nit: have you considered having these files saved to a temp dir so they have no chance of polluting the user's working directory (if something fails) and they will be automatically deleted by the OS? https://www.gnu.org/software/autogen/mktemp.html. May not be easy in js, but just wanted to toss out the suggestion!

Reply:
thats a really good idea, but im not sure how to do that in an OS agnostic way, but maybe its good enough to run in linux and mac. Implementation would be a bit tricky as well, but i do like the idea since there is a chance this test could fail and the user would not know to clean it up.

Reply:
I've updated readme to mention this folder exists, and how to clean it up